pagopa/pm/spec.json

```diff
@@ -1818,6 +1818,55 @@
           }
         ]
       }
+    },
+    "/v1/wallet/{idWallet}/payment-status": {
+      "put": {
+        "summary": "change the ability to pay",
+        "operationId": "changePayOption",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "idWallet",
+            "type": "integer",
+            "required": true,
+            "description": "Numeric ID of the wallet"
+          },
+          {
+            "in": "body",
+            "name": "payRequest",
+            "description": "New Status required",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/WalletPaymentstatus"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "New wallet State",
+            "schema": {
+              "$ref": "#/definitions/WalletV2Response"
```

Comment:
Is it right that the return type is `WalletV2Response`?  
Atm we have `Wallet`, `WalletV2` and `WalletV3`. What will happen when we update to `WalletV3` and the answer to this call fails the check?

Reply:
Yes it should be (see https://pagopa.atlassian.net/browse/PM-100?focusedCommentId=13990)
I don't know what could be happen, stay to the PM to respect these spec
Atm, as you said, we have a mix: e.g `actions/favourite` returns a `Wallet` https://github.com/pagopa/io-services-metadata/blob/c49aee534af9a33cea84c6dc3ec2910fba7fe941/pagopa/pm/spec.json#L1705